src/blindpsbt.cpp

```diff
@@ -476,6 +476,9 @@ BlindingStatus BlindPSBT(PartiallySignedTransaction& psbt, std::map<uint32_t, st
                         }
                     }
                 }
+                else {
```

Comment:
Prefer to have this as the if, with the existing code in the else. As it is its difficult to see what if this else relates to when reviewing. i.e.

```
                if (!blind_issuance) {
                    input_asset_blinders.emplace_back();
                } else {
```

Reply:
Also all of your `else` statements (here and elsewhere) are on new lines which is inconsistent with the rest of the code. Please use
```
} else {
```
Instead of
```
}
else {
```

Here and in the other changes.

Reply:
for code formatting/style wouldn't be better to rely on a GitHub action that enforces shared rules automatically?

Reply:
That would make the code harder to rebase over upstream core changes, unfortunately.